### DISCUSSION TITLE
How to change the color of a drawCircle inside a Graphics after adding it to the app.stage?

### DISCUSSION THREAD

**Discussion by silviubogan** [pixijs/pixijs#8221] [Q&A]
I currently am learning Pixi and I think I solved this by creating one circle of a color and another of some other color (although I am still working on the code), but I think it would be a faster way to be able to change the color of a Graphics somehow, although I don't know what would that do if inside the Graphics there would be multiple shapes to fill. I am sure this is not a new use case and I am wondering what is the common practice regarding this context.

**Comment by srpatel** [comment]
If you want to change the colour of all the shapes in the graphics at once, you can change the tint. You would need a new graphics instance for each shape you want to colour differently.

```
// Draw a white circle on a graphics instance
const g = new PIXI.Graphics();
g.beginFill(0xffffff);
g.drawCircle(0, 0, 100);
g.endFill();
this.addChild(g);

// Tint the graphics any colour your want, e.g. red
g.tint = 0xff0000;
```
